lmfit/minimizer.py

```diff
@@ -749,7 +749,9 @@ def emcee(self, params=None, steps=1000, nwalkers=100, burn=0, thin=1,
             `result.flatchain[parname]`. The ``lnprob`` attribute contains the
             log probability for each sample in ``chain``. The sample with the
             highest probability corresponds to the maximum likelihood estimate.
-
+        sampler_kwargs : dict, optional
+            Add any sampler keyword arguments here., Typically ``a``,
```

Comment:
This should be reworded. e.g. `Additional keyword arguments that can be used to construct the emcee.PTSampler or emcee.EnsembleSampler objects [1]_`